nav2_rviz_plugins/src/nav2_panel.cpp

```diff
@@ -409,8 +808,16 @@ Nav2Panel::onInitialize()
     [this](const action_msgs::msg::GoalStatusArray::SharedPtr msg) {
       navigation_goal_status_indicator_->setText(
         getGoalStatusLabel(msg->status_list.back().status));
-      if (msg->status_list.back().status != action_msgs::msg::GoalStatus::STATUS_EXECUTING) {
-        navigation_feedback_indicator_->setText(getNavToPoseFeedbackLabel());
```

Comment:
Shouldn't we still reset this to be blank at the end of the task?

Reply:
Oops! Yes! Nice catch!

Reply:
Currently, we do not show how long did the robot took to complete the entire waypoint following.. We just show the time taken for a specific waypoint and reset the time and move to the next. Maybe we need to have "total time taken" for showing how long the robot took to complete... 

Also I think it would be a nice idea for logging the time taken for each waypoint, which might be used for some sort of analysis later.. 

I think this should be handled as a separate issue..

Reply:
> Currently, we do not show how long did the robot took to complete the entire waypoint following.. We just show the time taken for a specific waypoint and reset the time and move to the next. Maybe we need to have "total time taken" for showing how long the robot took to complete...

That would be a good idea, but I think that should be done in a followup PR since that would involve editing the waypoint follower message definition further to provide that information 

> Also I think it would be a nice idea for logging the time taken for each waypoint, which might be used for some sort of analysis later..

I'm not sure that's the role of this plugin, but we could have the waypoint follower's results provide that information! Also another follow up PR thing so that we don't bloat this too much (and so this can get in to break apart the changes for easier reviewing)

Reply:
Sure! I can add it to the issue tracker! If that's okay!